freud/order.pyx

```diff
@@ -740,6 +745,67 @@ cdef class LocalQl(Compute):
     def __str__(self):
         return repr(self)
 
+    def _plot(self, mode, name, near, ax):
+        R"""Helper function for plotting.
+
+        Args:
+            mode (str): Plot mode that is an attribute of :code:`self`.
+            name (str): "Q" or "W".
+            near (bool): If true, add Near in title.
+            ax (:class:`matplotlib.axes.Axes`): Axis to plot on. If
+                :code:`None`, make a new figure and axis.
+                (Default value = :code:`None`)
+
+        Returns:
+            (:class:`matplotlib.axes.Axes`): Axis with the plot.
+        """
+        import plot
+        plot_data = getattr(self, mode)
+        mode = mode.replace(name, "Q")
+        if mode == "Ql":
+            mode_str = ""
+        elif mode == "ave_Ql":
+            mode_str = "Average"
+        elif mode == "norm_Ql":
+            mode_str = "Normalized"
+        elif mode == "ave_norm_Ql":
+            mode_str = "Average Normalized"
+        else:
+            raise RuntimeError("Invalid plot moode.")
+        title = r"Local ${}_{{{}}}$".format(name, self.sph_l)
+        if near:
+            title += " Near"
+        xlabel = r"{} ${}_{{{}}}$".format(mode_str, name, self.sph_l)
+        return plot.histogram_plot(plot_data,
+                                   title=title,
+                                   xlabel=xlabel,
+                                   ylabel=r"Number of particles",
+                                   ax=ax)
+
+    @Compute._computed_method(("compute", "computeNorm",
+                               "computeAve", "computeAveNorm"))
+    def plot(self, mode, ax=None):
```

Comment:
Change the signature to `plot(self, ax=None, mode=None)` (swapped order).
Then
```python
if mode is None:
    mode = self.plot_mode
```
and update the docstring to say it defaults to the most recently computed attribute.

Reply:
And similarly for the other classes.